rpcs3/rpcs3qt/gui_settings.cpp

```diff
@@ -225,10 +243,8 @@ bool gui_settings::GetBootConfirmation(QWidget* parent, const gui_save& gui_save
 		else if (gui_save_entry == gui::ib_confirm_fw)
 		{
 			title = tr("Missing Firmware Detected!");
-			message = tr("Install the PS3 Firmware (Menu: File -> Install Firmware)."
-				"\n<br>For more information read the <a href=\"https://rpcs3.net/quickstart\">quickstart guide</a>."
-				"\nCommercial games do not work without firmware! Do you wish to continue!?");
-			icon = QMessageBox::Critical;
+			message = tr("Installing firmware (PS3UPDAT.PUP) file is required for commercial games."
```

Comment:
This sentence sounds weird. Idk, maybe improve it somehow.

Reply:
Reworded.